contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -513,4 +513,22 @@ public void testGetLogicalProcessorInformationEx() {
             assertTrue(cache.associativity == WinNT.CACHE_FULLY_ASSOCIATIVE || cache.associativity > 0);
         }
     }
+
+    public void testSetCurrentProcessPriority() {
+        Kernel32Util.setCurrentProcessPriority(Kernel32.HIGH_PRIORITY_CLASS);
+    }
+
+    public void testSetCurrentThreadPriority() {
+        Kernel32Util.setCurrentThreadPriority(Kernel32.THREAD_PRIORITY_ABOVE_NORMAL);
```

Comment:
You could add `GetThreadPriority` and similar and check that this actually worked.